py3status/modules/google_calendar.py

```diff
@@ -0,0 +1,507 @@
+        strftime directives for dates.
+        (default '%a %d-%m')
+    format_event: The format for each event. The information to display for each event
+        can be toggled with 'button_toggle' based on the value of 'is_toggled'.
+        (default  '[\?color=event {summary}][\?if=is_toggled  ({start_time}
+         - {end_time}, {start_date})|[\?if=location  ({location})] [{format_timer}]]'
+    format_notification: The format for event warning notifications.
+        (default '{summary} {start_time} - {end_time}')
+    format_separator: The string used to separate individual events.
+        (default '\?color=separator \|')
+    format_time: The format for time related format placeholders (except {format_timer}).
+        May use any Python strftime directives for times.
+        (default '%I:%M %p')
+    format_timer: The format used for the {format_timer} placeholder to display time
+        until an event starts or time until an event in progress is over based.
+        (default '\?color=time ([\?if=days {days}d ]
+        [\?if=hours {hours}h ][\?if=minutes {minutes}m]) [\?if=is_current left]')
+    ignore_all_day_events: Sets whether to display all day events or not.
+        (default False)
+    num_events: The maximum number of events to display.
+        (default 3)
+    thresholds: Thresholds for events. The first entry is the color for event 1,
+        the second for event 2, and so on.
+        (default [])
+    time_to_max: Threshold (in minutes) for when to display the {format_timer}
+        string; e.g. if time_to_max = 60, {format_timer} will only be displayed
+        for events starting in 60 minutes or less.
+        (default 180)
+    warn_threshold: The number of minutes until an event starts before a warning is
+        displayed to notify the user; e.g. if warn_threshold = 30 and an event is
+        starting in 30 minutes or less, a notification will be displayed. disabled (= 0)
+        by default.
+        (default 0)
+    warn_timeout: The number of seconds before a warning should be displayed again.
+        (default 300)
+
+Control placeholders:
+    {is_toggled} a boolean toggled by button_toggle
+
+Format placeholders:
+    {events} All the events to display.
+
+format_event and format_notification placeholders:
+    {description} The description for the calendar event.
+    {end_date} The end date for the event.
+    {end_time} The end time for the event.
+    {location} The location for the event.
+    {start_date} The start date for the event.
+    {start_time} The start time for the event.
+    {summary} The summary (i.e. title) for the event.
+    {format_timer} The time until the event starts (or until it is over
+        if already in progress).
+
+format_timer placeholders:
+    {days} The number of days until the event.
+    {hours} The number of hours until the event.
+    {minutes} The number of minutes until the event.
+
+Color options:
+    color_separator: Color for separator.
```

Comment:
(Imo) This option is not worth keeping... as users can just do...
```
format_separator = '\?color=#ffccff \|
```
instead of...
```
format_separator = '\?color=separator \|   # sometimes it can be in user's config.
color_separator = '#ffccff'
```